source/renderer/app/containers/loading/LoadingPage.js

```diff
@@ -26,12 +26,13 @@ export default class LoadingPage extends Component<InjectedProps> {
   }
 
   get isSystemTimeError() {
```

Comment:
@disassembler Do we need this method? Maybe we can just remove method and also this: https://github.com/input-output-hk/daedalus/blob/719ee1bcbc4e1824797ee533d4a1bd55c6448c96/source/renderer/app/containers/loading/LoadingPage.js#L20

Reply:
No we can't as that would mean the component is nowhere used and then we would lose the translations once we run the translation manager @tomothespian